lib/ParamedicTargetChooser.js

```diff
@@ -134,11 +134,11 @@ class ParamedicTargetChooser {
         logger.info('cordova-paramedic: Choosing Target for iOS');
 
         const simulatorModelId = utilities.getSimulatorModelId(this.cli, target);
```

Comment:
```suggestion
        const simulatorModelId = utilities.getSimulatorModelId(this.cli, target).trim();
```

Currently there is still a newline at the end, which works fine with the actual command that uses it later but is not pretty or useful of course. (Was there previously as well, see deleted line 138)

Reply:
I am confused on what you are saying here.